### PR TITLE
Update feature table

### PR DESCRIPTION
**Pull request by hongdai**
* Update known issue link to the WCF release note.
* Simplify SPN and UPN columns. The detail has been added to the acutal issue.

### CODE REVIEW

File: release-notes/SupportedFeatures-v1.0.0.md [hunk 1]
```diff
@@ -34,23 +34,22 @@ Supported features
...
 |Client Authentication|Basic           |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
-|              |Digest                 |:white_check_mark:                   |:white_check_mark:                  |[:no_entry_sign:](#known_issues)           | [:no_entry_sign:](#known_issues)    |
-|              |NTLM                   |:white_check_mark:                   |:white_check_mark:                  |[:no_entry_sign:](#known_issues)           | [:no_entry_sign:](#known_issues)    |
-|              |Kerberos/TCP           |[:no_entry_sign:](#known_issues)     |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
-|              |Kerberos/HTTP          |:white_check_mark:                   |:white_check_mark:                  |[:no_entry_sign:](#known_issues)           | [:no_entry_sign:](#known_issues)    |
+|              |Digest                 |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :grey_question:|
```
Comment (zhenlan) [inline release-notes/SupportedFeatures-v1.0.0.md]: Based on #1332, I think we can now mark digest not working on Linux and OS X.

File: release-notes/SupportedFeatures-v1.0.0.md [hunk 2]
```diff
@@ -19,7 +19,7 @@ Supported features
 |Transports    |Http                   |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
 |              |Https                  |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:           |
 |              |Tcp                    |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:          |
-|              |WebSockets             |[:warning:](#known_issues)           |:white_check_mark:                  |[:no_entry_sign:](#known_issues)           | [:no_entry_sign:](#known_issues)    |
```
Comment (zhenlan) [inline release-notes/SupportedFeatures-v1.0.0.md]: I cannot add a comment there... can we remove line 8, 10 and 11 (the tip and :grey_question:) as they are not much useful or not used anymore?

File: release-notes/SupportedFeatures-v1.0.0.md [hunk 3]
```diff
@@ -34,23 +34,22 @@ Supported features
...
+|              |NTLM                   |:white_check_mark:                   |:white_check_mark:                  |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)           | [:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)    |
+|              |Kerberos/TCP           |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)     |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
+|              |Kerberos/HTTP          |:white_check_mark:                   |:white_check_mark:                  |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)           | [:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)    |
 |              |Certificate/TCP       |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
 |              |Certificate/HTTP      |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :grey_question:                     |
```
Comment (zhenlan) [inline release-notes/SupportedFeatures-v1.0.0.md]: Do we have an answer for Certificate/HTTP on Mac now?

File: release-notes/SupportedFeatures-v1.0.0.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ Supported features
 |Transports    |Http                   |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
```
Comment (zhenlan) [inline release-notes/SupportedFeatures-v1.0.0.md]: Can we add a link pointing back to the release notes underneath the title (line 3) like what we had for [RC2](https://github.com/dotnet/wcf/blob/master/release-notes/SupportedFeatures-v1.0.0-rc2.md#wcf-features-in-rc2)?

File: release-notes/SupportedFeatures-v1.0.0.md [hunk 5]
```diff
@@ -34,23 +34,22 @@ Supported features
...
 |              |ChannelBase\<T\>       |:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
 |Extensibility |IClientMessageInspector|:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
 |              |IClientMessageFormatter|:white_check_mark:                   |:white_check_mark:                  |:white_check_mark:                         | :white_check_mark:                  |
-|Management    |ETW Tracing            |[:no_entry_sign:](#known_issues)      |:white_check_mark:                  |[:no_entry_sign:](#known_issues)          |[:no_entry_sign:](#known_issues)     |          
+|Management    |ETW Tracing            |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)      |:white_check_mark:                  |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)          |[:no_entry_sign:](https://github.com/dotnet/wcf/releases/tag/v1.0.0)     |          
 
 
 #### Platforms
```
Comment (zhenlan) [inline release-notes/SupportedFeatures-v1.0.0.md]: NIT: line 58, "... include..."
Also can we update Linux distro names to match the names in dotnet/core#142 (it's okay to skip versions but match the full name and casing)?

File: release-notes/SupportedFeatures-v1.0.0.md (hunk 3 above)
Comment (hongdai) [inline release-notes/SupportedFeatures-v1.0.0.md]: Haven't figured out a way to test it.

File: release-notes/SupportedFeatures-v1.0.0.md (hunk 4 above)
Comment (hongdai) [inline release-notes/SupportedFeatures-v1.0.0.md]: Will give it a try.

File: release-notes/SupportedFeatures-v1.0.0.md (hunk 5 above)
Comment (hongdai) [inline release-notes/SupportedFeatures-v1.0.0.md]: Agreed.

File: release-notes/SupportedFeatures-v1.0.0.md (hunk 1 above)
Comment (hongdai) [inline release-notes/SupportedFeatures-v1.0.0.md]: Digest domain test passed on Linux when the server machine is domain joined. Linux machine can communicate with Redmond domain joined test server, OSX cannot.

It's unknown why domain joined server does not hit issue  #1332.

File: release-notes/SupportedFeatures-v1.0.0.md (hunk 2 above)
Comment (hongdai) [inline release-notes/SupportedFeatures-v1.0.0.md]: I'll remove them.

File: release-notes/SupportedFeatures-v1.0.0.md [hunk 6]
```diff
@@ -34,28 +32,27 @@ Supported features
...
 &ensp;&ensp;For more details about what UWP is please see this [dotnet Glossary doc](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/glossary.md).
 
-&ensp;&ensp;Linux OSs includes centos, debian, fedora, linux, opensuse, rhel and ubuntu.
+&ensp;&ensp;Linux OSs includes Red Hat Enterprise Linux, CentOS, Debian, Fedora, Linux Mint, OpenSUSE, Oracle Linux and Ubuntu.
 
 &ensp;&ensp;For supported versions of each platform and other details please see the [.NET Core 1.0.0 Release Notes](https://github.com/dotnet/core/blob/master/release-notes/1.0/Release-Notes-1.0.0.md).
```
Comment (zhenlan) [inline release-notes/SupportedFeatures-v1.0.0.md]: The link was changed to https://github.com/dotnet/core/blob/master/release-notes/1.0/1.0.0.md

Comment (hongdai) [inline release-notes/SupportedFeatures-v1.0.0.md]: Fixed.

**Comment by zhenlan** [comment]
LGTM. Thanks @hongdai.

**Pull request merged**